velox/exec/tests/utils/OperatorTestBase.h

```diff
@@ -150,4 +151,23 @@ class OperatorTestBase : public testing::Test,
   // Used for IO prefetch and spilling.
   std::unique_ptr<folly::IOThreadPoolExecutor> ioExecutor_;
 };
+
+/// Test utility class for HashProbe operator.
+class HashProbeTestHelper {
+ public:
+  HashProbeTestHelper(Operator* op) : op_(static_cast<HashProbe*>(op)) {
+    VELOX_CHECK_NOT_NULL(op_);
+  }
+
+  bool hasInput() const {
+    return op_->input_ != nullptr;
+  }
+
+  bool hasInputSpiller() const {
+    return op_->inputSpiller_ != nullptr;
+  }
+
+ private:
+  exec::HashProbe* const op_;
+};
```

Comment:
Do we really need this class? Can we just put the individual checks in the code directly? And then use 
VELOX_TEST_FRIEND to declare that friend class?